usefulUtilities/sit/sitClient.js

```diff
@@ -543,24 +543,25 @@
     // Entity methods
     SitClient.prototype = {
         remotelyCallable: [
-            "createSittableUI",
-            "deleteSittableUI",
-            "startSitDown",
-            "check"
+            "createClickToSitOverlay",
+            "deleteClickToSitOverlay",
+            "checkBeforeSitDown",
+            "heartbeatRequest",
+            "startSitDown"
         ],
         // Zone methods
```

Comment:
No more zones on sit cubes so we can remove the comment

Reply:
ack